Customer Story: Purplewave (#6794)
PostHog/posthog.com: contents/customers/purplewave.md

---
title: Why PurpleWave moved from HotJar and Checkmarket, to PostHog
customer: Purplewave
logo: ../images/customers/purplewave/purplewave_logo.png
logoDark: ../images/customers/purplewave/purplewave_logo_dark.png
featuredImage: ../images/customers/purplewave/purplewave.jpg
industries:
    - ECommerce
users:
    - Product
toolsUsed:
    - Surveys
date: 2023-09-15
---

> [Purplewave](https://www.purplewave.com/) is the world’s leading two-way marketplace for heavy duty equipment such as construction and agriculture fleets. The team of 220 people maintains a platform trusted by 4.75m annual visitors, with more than 150 fleet-scale auctions each year. 

Reviewer: ```suggestion
[Purplewave](https://www.purplewave.com/) is the world’s leading two-way marketplace for heavy duty equipment, such as construction and agriculture fleets. The team of 220 people maintains a platform trusted by 4.75m annual visitors, with more than 150 fleet-scale auctions each year. 
```

Now, the team plans to 3x that growth — something it can only do by adopting more scalable systems for collecting and handling user feedback about the platform. 

“Previously a lot of teams operated on gut decisions and intuition,” says Purplewave’s Product Owner, Matt Amick. “We had data, but it wasn’t mobilized well because it was accessed through Tableau, which isn’t the easiest to use. We started looking around for something that could help us make decisions based on data — and that’s when we found PostHog!”

> “A lot of the other companies we spoke to were using tools like [Amplitude](/blog/posthog-vs-amplitude), but they all said they try to get you in and then charge you a million dollars once you’re set up...Our team is all about just being honest and not hiding stuff, so we preferred PostHog.”

Reviewer: ```suggestion
“A lot of the other companies we spoke to were using tools like [Amplitude](/blog/posthog-vs-amplitude), but they all said they try to get you in and then charge you a million dollars once you’re set up... Our team is all about just being honest and not hiding stuff, so we preferred PostHog.”
```

Purplewave began using many PostHog features, including [analytics](/product-analytics), [flags](/feature-flags), and [replays](/session-replay) — and when user surveys launched in their first beta state, the team adopted that too.

<BorderWrapper>
<Quote
    imageSource="/images/customers/purplewave_matt.jpg"
    size="md"
    name="Matt Amick"
    title="Product Owner, Purplewave"
    quote={`“I love everything about PostHog from the design to the culture, just everything. When the team launched the surveys beta and we could make no-code surveys and have everything in one place too? That was just phenomenal.”`}
/>
</BorderWrapper>

### How Purplewave reached a 25% response rate for user surveys

> Before switching to PostHog, Purplewave had previously attempted to solicit feedback over email using CheckMarket by Medallia. Unfortunately, CheckMarket’s approach introduced needless friction for users and led to a poor experience that was reflected in low response rates. 
> 
> “We used CheckMarket for our [NPS surveys](/tutorials/nps-survey) but were only getting a 14% response rate,” says Matt. “So, I decided to run a test with in-app surveys using PostHog and roll that out to identified, registered users only. The response rate then was 25%, which was almost twice as good!”
> 
> In addition to improving the response rate, conducting surveys with PostHog was also ideal because it centralized data into a single platform. Responses could be analyzed using PostHog insights, linked to feature flags, or observed with session replays — none of which was true with other tools the team had tried. 
> 
> “We had used [HotJar](/blog/posthog-vs-hotjar) before and I just hated having responses in a separate software,” says Matt. “I want it all in one platform so it’s easy to access — because ultimately everything needs to come back to the [data warehouse](/docs/data-warehouse). You have to have your data centralized, and with HotJar it wasn’t.” 
> 
> 
> ![PostHog surveys](../images/customers/purplewave/purplewave_survey.png)

Author: ```suggestion
Before switching to PostHog, Purple Wave had previously attempted to solicit feedback over email using other tools. Response rates were poor, leading the team to consider more innovative ways to collect feedback. 

“We were only getting a 14% response rate on NPS surveys previously,” says Matt. “So, I ran a test with in-app surveys using PostHog. I was still able to target identified users, and the response rate jumped to 25% — almost twice as good!”

Conducting surveys with PostHog also enabled Purplewave to centralize data into a single platform. Responses could be analyzed using insights, linked to feature flags, or observed with session replays — all in one place. 

“I really hate having to switch between separate software,” says Matt. “I want it all in one platform so it’s easy to access. Ultimately everything comes back to the [data warehouse](/docs/data-warehouse). You have to have your data centralized in order to use it effectively.” 

![PostHog surveys](../images/customers/purplewave/purplewave_survey.png)
```
